text/0000-rfc-issue-generation.md

```diff
@@ -0,0 +1,196 @@
+Later, after the RFC has been approved, it requires maintainers of affected sub-teams to create issues on their respective repositories.
+This typically happens days, weeks, if not months after the maintainers might have reviewed the RFC.
+In addition, it usually requires the person merging the RFC to hunt down (or ping) any maintainers for sub-teams that haven't created relevant issues.
+
+The goal of this RFC is to minimize the overhead of creating issues for maintainers and the individual merging the RFCs by incoorporating the processes into the review process as opposed to the merge process.
+
+# What it is
+[what-it-is]: #what-it-is
+
+This solution is comprised of a _bot_ (a faux GitHub account) and updates to the [`merge-rfc.sh`](https://github.com/buildpacks/rfcs/blob/main/merge-rfc.sh) script. The bot, likely to be implemented via GitHub Actions, monitors pull requests in `buildpacks/rfcs` for comments while maintaining a list of _queued issues_. `merge-rfc.sh` would be updated to parse the list of queued issues and create those issues in their respective repositories.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+When a pull request is created on `buildpacks/rfcs` the bot would post a comment with the following content:
+
+>
+> Maintainers,
+>
+> As you review this RFC please queue up issues to be created using the following commands:
+>
+>  - `/queue-issue <repo> "<title>" [labels]...`
+>  - `/unqueue-issue <uid>`
+>
+> **Queued Issues**
+> 
+> _(none yet)_
+>
+
+The bot will monitor comments to pull requests. It will parse the contents in search for instructions `/queue-issue` or `/unqueue-issue`. If found, it would take the approprite action by updating the initial comment posted with a list of queued issues. In the case of an addition a short UID would be generated.
+
+#### Example Addition
+
+Addition Comment:
+
+```
+/queue-issue buildpacks/rfcs "Create issues from RFCs (merge-rfc.sh)" [label1][label2]
+```
+
+Updated Bot Comment:
+
+>
+> Maintainers,
+>
+> As you review this RFC please queue up issues to be created using the following commands:
+>
+>  - `/queue-issue <repo> "<title>" [labels]...`
+>  - `/unqueue-issue <uid>`
+>
+> **Queued Issues**
+> 
+> - Xoq7ID - buildpacks/rfcs "Create issues from RFCs (merge-rfc.sh)" [label1][label2]
+>
+
+
+#### Example Removal
+
+Remove Comment:
+
+```
+/unqueue-issue Xoq7ID
+```
+
+Updated Bot Comment:
+
+>
+> Maintainers,
+>
+> As you review this RFC please queue up issues to be created using the following commands:
+>
+>  - `/queue-issue <repo> "<title>" [labels]...`
+>  - `/unqueue-issue <uid>`
+>
+> **Queued Issues**
+> 
+> _(none)_
+>
+
+Once the RFC is merged via the [`merge-rfc.sh`](https://github.com/buildpacks/rfcs/blob/main/merge-rfc.sh) script, the script could detect issues by looking through the top comment from the _bot_ and creating any queued issues. Some additions that the script may do but subject to change:
```

Comment:
I assume this could be done via a GitHub action on PR merges?

Reply:
At a later point in time or via separate initiative, yes.